### PR TITLE
[PIE-673][Martin] added IL Dashboard definitions

### PR DESCRIPTION
**Pull request by MartinM1312**
## 💅🏼 What issue does this fix?
Added definitions for IL Dashboard

## 🍂 Before You Submit

* [X] Did you write & run tests and linters?
* [ ] Did you run Google Lighthouse and/or WebAIM (Wave) on UI components in your PR?
* [ ] Does your PR contain any required translations?
* [ ] Are your primary keys UUIDs on any new tables?

## 🛷 Deployment

* [ ] Migrations
* [ ] Dependencies

### CODE REVIEW

File: client/src/Dashboard/DashboardDefinitions.js [hunk 1]
```diff
@@ -1,107 +1,23 @@
 import React from 'react'
 import PropTypes from 'prop-types'
-import { Button, Collapse, Tag } from 'antd'
-import { ArrowUpOutlined } from '@ant-design/icons'
-import { useTranslation } from 'react-i18next'
+import NEDashboardDefintions from './NEDashboardDefinitions'
```
Comment (fabianraf) [inline client/src/Dashboard/DashboardDefinitions.js:3]: The typo remains.

**Review by fabianraf — CHANGES_REQUESTED** [review]

File: client/src/i18n/locales/es/index.json [hunk 2]
```diff
@@ -197,6 +197,11 @@
   "neAuthorizedPeriodDef": "Se trata de la fecha de inicio y de finalización para la que se autoriza al niño a recibir subsidios para el cuidado infantil. Puedes encontrar esta información en la carta de autorización.",
   "neHoursRemainingDef": "El número de horas que el niño aún tiene disponibles para asistir. Puedes encontrar el número total de horas autorizadas en la carta de autorización.",
   "neFullDaysRemainingDef": "Cuántas unidades diarias tiene todavía el niño para asistir. Puedes encontrar el número total de unidades diarias autorizadas en la carta de autorización.",
+  "ilAttendanceRateDef": "Revisar cuáles niños están en camino para asistir 70% y por ende para reciir el pago completo del subsidio",
```
Comment (fabianraf) [inline client/src/i18n/locales/es/index.json:200]: Typo in "reciir". Should be "recibir"

File: client/src/Dashboard/ILDashboardDefinitions.js [hunk 3]
```diff
@@ -0,0 +1,100 @@
+import React from 'react'
```
[the remark this answers is not in the document]
Comment (MartinM1312) [reply client/src/Dashboard/ILDashboardDefinitions.js:1]: > Please add tests for this component

I've already created I think you missed it

File: client/src/i18n/locales/es/index.json [hunk 4]
```diff
@@ -197,6 +197,11 @@
...
+  "ilAtRiskDef": "El niño ha estado ausente frecuentemente, y es posible que no se reciba el pago completo del subsidio. Comunicate con la familia para averiguar si podría asistir más días",
```
Comment (fabianraf) [inline client/src/i18n/locales/es/index.json:201]: "Comunícate"

**Review by fabianraf — APPROVED** [review]

**Review by fabianraf — APPROVED** [review]

**Pull request merged**